### DISCUSSION TITLE
Backspace doesn't work with programs that create a custom GUI

### DISCUSSION THREAD

**Discussion by cullendoran** [cc-tweaked/CC-Tweaked#1738] [Q&A] [answered]
So for some reason I can't seem to use the backspace key on programs such as Turtle Architect by CometWolf where it creates it's own GUI, not sure if it's a problem with windows keyboard types or what but I can't use programs like I mentioned before because of this.

**Comment by fatboychummy** [comment, marked as answer]
This issue should be reported on the [TurtleArchitect repository](https://github.com/CometWolf/TurtleArchitectV2/) instead.

https://github.com/CometWolf/TurtleArchitectV2/blob/0e0f2672a48adae6868598ad9e2677adef1c55f7/TAFiles/EventHandlers/main.Lua#L52

They hard-coded key values instead of using the constants in the `keys` library. The actual values can change between minecraft versions, but the `keys` library constants update to reflect this. Because they hard-coded the constants, their program will only work on one version of minecraft -- the version they originally created it on.

**Reply by cullendoran** [reply]
Ah ok sorry, thought it was a issue with keyboard settings, I will go there now
